artifacts/deploy/karmada-metrics-adapter.yaml

```diff
@@ -67,7 +67,7 @@ spec:
       volumes:
         - name: karmada-certs
           secret:
-            secretName: karmada-cert-secret
+            secretName: karmada-cert
```

Comment:
The secret doesn't seem to exist.

Reply:
That is the secret name if you install using karmadactl init
kubectl get secret -n karmada-system
NAME                   TYPE     DATA   AGE
etcd-cert              Opaque   4      23h
karmada-cert           Opaque   16     23h
karmada-webhook-cert   Opaque   2      23h
kubeconfig             Opaque   1      23h

Reply:
Ask @chaosi-zju to help take a look.
cc @chaosi-zju

Reply:
> the helm install needs more tweaks to the metrics adapter, but I guess the helm charts are more outdated as I couldn't make those work

It's weird, in fact we have the [CI](https://github.com/karmada-io/karmada/blob/master/.github/workflows/lint-chart.yaml) for the helm installation. Can you describe the problem you're experiencing?